ContinuousRegistration/Source/datasets.py

```diff
@@ -521,6 +557,9 @@ def __init__(self, input_directory, output_directory, max_number_of_registration
 
         self.file_names = take(sort_file_names(file_names), max_number_of_registrations // 2)
 
-
     def evaluate(self, superelastix, file_names, output_directory):
         self.evaluate_point_set(superelastix, file_names, output_directory)
+
+
+class Histology(Dataset):
```

Comment:
Do you have a "code name" for your data set? We might add other histology data sets in the future

Reply:
Okay great! Probably better now that later, so that the arguments (e.g. --histology-input-directory) would have to be changed in the future :)